OktaLogger.podspec

```diff
@@ -41,7 +41,7 @@ Pod::Spec.new do |s|
       instabugLogger.ios.source_files = [
         'Sources/OktaLogger/InstabugLogger/*'
       ]
-      instabugLogger.ios.dependency 'Instabug', '~> 10.7'
+      instabugLogger.ios.dependency 'Instabug', '~> 11.0.1'
```

Comment:
Do we need to pin with `=` exact version of Instabug? Their SDK is not always reliably works so I would avoid auto updates even to minor versions :)